common/changes/@itwin/cra-template-desktop-viewer/jake-template-update_2023-07-19-15-21.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/cra-template-desktop-viewer",
+      "comment": "Update to React 18, see [their changelog](https://github.com/facebook/react/blob/main/CHANGELOG.md)",
+      "type": "patch"
```

Comment:
Should be minor?

Reply:
My thought process was that we just bumped the minor when we merged in the react 18 upgrade for the rest of the repo, so this is just more along the same line

Reply:
Not quite following. Wouldn't "more along the same line" mean this should be minor as well?

Reply:
Changed to minor